app/src/main/res/values/dimens.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="dp_55">55dp</dimen>
-
+    <dimen name="dp_145">145dp</dimen>
+    <dimen name="dp_1">1dp</dimen>
+    <dimen name="dp_30">30dp</dimen>
+    <dimen name="sp_20">20sp</dimen>
+    <dimen name="dp_20">20dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_20` appears to be unused
 Unused resources make applications larger and slow down builds.